dags/main_summary.py

```diff
@@ -129,6 +129,21 @@
                        uri="https://raw.githubusercontent.com/mozilla/experiments-viewer/master/notebooks/import.py",
                        dag=dag)
 
+t12 = EMRSparkOperator(task_id="search_dashboard",
+                       job_name="Search Dashboard",
+                       execution_timeout=timedelta(hours=3),
+                       instance_count=5,
+                       owner="harterrt@mozilla.com",
+                       email=["telemetry-alerts@mozilla.com", "harterrt@mozilla.com"],
+                       env=mozetl_envvar("search_dashboard", {
+                             "start_date": "{{ ds_nodash }}",
```

Comment:
This should be `submission_date` to reflect the seachdb script, or the script should change the option to `start_date`.